packages/nextjs/components/Header.tsx

```diff
@@ -55,7 +87,7 @@ export const Header = () => {
   );
 
   return (
-    <div className="sticky lg:static top-0 navbar bg-primary min-h-0 flex-shrink-0 justify-between z-20 shadow-md shadow-secondary">
+    <div className="sticky   lg:static   top-0 na vbar bg-primary min-h-0 flex-shrink-0 justify-between z-20 s h adow-md shadow-secondary">
```

Comment:
During my tests the header was loading in a very strange way, I think some spaces got added there and made it go crazy :)

![challenge-0-menu](https://github.com/scaffold-eth/se-2-challenges/assets/55535804/5d00add9-9a6c-46df-b4a6-693bda4143ba)

Reply:
Oops my bad, tysm 🙌 fixed at [844ec01](https://github.com/scaffold-eth/se-2-challenges/pull/5/commits/844ec014f4537c463e5e18728be9514b93c69e8b)